### PR TITLE
Make new desigin for few views

### LINKED ISSUE #224: Poprawić szatę graficzną

### PR DESCRIPTION
**Pull request by Derstilon**
New "Create session" view design
New "List" view design
New "Create quiz" view design
New "Create question" view design
New color palette

### CODE REVIEW

File: src/lecturer/components/app/App.tsx [hunk 1]
```diff
@@ -23,14 +23,24 @@ import TopBar from "../topBar/topBar";
...
-            main: "#80A3E4",
-            dark: "#4870AC",
+
+            main: "#4C3957",
+
+            // main: "#59679F",
+            // light: "#95A0C7",
+
+            // light: "#ABA685",
+            // main: "#3A3D29",
```
Comment (ostatni5) [inline src/lecturer/components/app/App.tsx]: Remove comments

File: src/lecturer/components/app/App.tsx [hunk 2]
```diff
@@ -23,14 +23,24 @@ import TopBar from "../topBar/topBar";
...
-            main: "#D9A21B",
-            dark: "#877455",
+
+            main: "#41658A",
+
+            // main: "#4B9569",
+            // light: "#77BB93",
+
+            // light: "#A67463",
+            // main: "#59170E",
```
Comment (ostatni5) [inline src/lecturer/components/app/App.tsx]: Remove comments

File: src/lecturer/components/createQuizView/CreateQuizList.tsx [hunk 3]
```diff
@@ -24,6 +24,8 @@ interface QuizListViewProps {
 
     titleQuiz?: string;
 
+    cardClass?: string;
+
```
Comment (ostatni5) [inline src/lecturer/components/createQuizView/CreateQuizList.tsx]: why enters

**Review by ostatni5 — APPROVED** [review]
Beautiful colors 
a

**Review by rozchlastywacz — APPROVED** [review]
11/10

**Pull request merged**